src/hooks/register_events.js

```diff
@@ -152,7 +160,7 @@ const hook = async function (options) {
     return
   }
 
-  if (!['app:deploy', 'app:undeploy'].includes(options.Command.id)) {
+  if (!['app:deploy', 'app:undeploy', 'app:run'].includes(options.Command.id)) {
     aioLogger.debug('App builder extension plugin works only for app:deploy command. Skipping...')
```

Comment:
Probably we should update wording of debug message to list all commands that are handled by plugin